packages/pipeline/src/parsers/events/exchange_events.ts

```diff
@@ -1,11 +1,13 @@
 import { ExchangeCancelEventArgs, ExchangeCancelUpToEventArgs, ExchangeFillEventArgs } from '@0x/contract-wrappers';
 import { assetDataUtils } from '@0x/order-utils';
-import { AssetProxyId, ERC721AssetData } from '@0x/types';
+import { AssetProxyId, ERC721AssetData, SingleAssetData } from '@0x/types';
 import { LogWithDecodedArgs } from 'ethereum-types';
 import * as R from 'ramda';
 
 import { ExchangeCancelEvent, ExchangeCancelUpToEvent, ExchangeFillEvent } from '../../entities';
-import { bigNumbertoStringOrNull } from '../../utils';
+import { bigNumbertoStringOrNull, convertAssetProxyIdToType } from '../../utils';
+
+// tslint:disable:no-unnecessary-type-assertion
```

Comment:
Again, I would prefer if we didn't disable this linter rule for the entire file.